setup.py

```diff
@@ -22,57 +22,57 @@
         #   tensorboard needs >=41
         #   50.0 is broken: https://github.com/pypa/setupatools/issues/2353
         "setuptools>=41.0.0,!=50.0",
-        "cached-property",
-        "click",  # used in scl
+        "cached-property==1.5.2",
+        "click==8.0.3",  # used in scl
```

Comment:
This hard lock of the dependencies seems overly harsh. This does improve the stability of smarts but it also locks us out of improvements provided by newer packages. I think we should go with `>=` rather than `==`.

Reply:
Changed `==` to `>=` for packages where versions were newly specified.